contrib/fonts-nerd-fonts/template.py

```diff
@@ -0,0 +1,208 @@
+    "7ab613fac9d5931bd32d8e5a438ae0cb17536317e171f28ee5ddb8a171941041",
+    "89b9023f3f6d5e4b8040022ff29b36faee440cf49015fda163215c922569b5f3",
+    "4fb0ff16a5e587e65f59e55d08283729de700f0c0bf682ec1bef18ab51264ada",
+    "23552caf1b6075858756de2e8f3a3fbb9799b478882f3f7afeefa8ca7f4601ae",
+    "85b0b39b30a9ddd89462b0a7a732b461de04f667dc01a5e987bf0c2accbb576f",
+    "2c9f5a1117e1b5a640287081680837e7b870c94868627766483011ff0b80c3f7",
+    "69e0a42a6e8072b2a084f09a5056e857e9b620afd57d7d07c8d1da2cf4809b41",
+    "7a96f8cc30e57a426ffce3b844deb10d6325181b28df9b76735308f2d787dc2d",
+    "a016b61e49c395c5b1e5bd2c919ae668724a33f2ef14da3becf932da401a080d",
+    "1be123fa355a9acee79d19891c02499792f6a4e12ffa8973c6b1cb738465f337",
+    "b5d5e7b26c76f1cfe97a697716fdd682b18faf9f73883915c2b0c2d9fcc48cf5",
+    "3132ff085ce404a3c065054638f3dc68de27d07a5cc5379b90b2a248877971eb",
+    "6cf8822bc1ca18e34b06578c7499f380c019e6ffc883eed26df5f498dfcc4006",
+    "2282cb16651a377d5b084486a268189fdafd9067775f1f24ffaf5a36a2459501",
+    "9d0ad55cbd83967ca93e940f4951d41b23a44c892d9a4788af460d71f8bb81a4",
+    "a2720d1fbf7ebb51b6a0ab191de41d96c7594d61f02c582db2249ea30c1ce263",
+    "62484ea8f66ae3b75610b89c5d3258f8610844fec6d8402239c27a5042dccc7d",
+    "1abdaf5542b75a51d75de6373148a8e401b28eda88e6c90f243fb6114fd659ea",
+    "e5d2e8180d5df1fe473b598edac5f67ce38b1d906248936eb386dd332a338346",
+    "c7a5ff556ff4aad5381e8a57e0d17d79c591e1267f0a9c271b09306ac4581aba",
+    "eb3697dd13c5e50951023c0768538a2f70e1e4f22d94800a45779a6d1b49f7a3",
+    "5c60700788b093e0bfb30b2f59b52397f3fdc2cdf2b6fb9307a58b61e7ddfe74",
+    "4be621219ff5508c4176487b4fdc8cd0f6d8d8fceeb935f83df11a49dca3096f",
+    "96d43a8c53173b992537a1d34ae984f65a8999b7c8991a02c17d4b77497581fa",
+    "fcb796892b105c98ad05a29f184f09a02038970ff6a95f8cda082cd1b1b08cde",
+    "fe3bd035d43b943de17e6e0da1dfb0bd1dcc1b319e10c5e5925ab70436d48385",
+    "1a58c57b63f380441ea07d44780680844afe12dd299651b87db59f9f311702d1",
+    "e5f7db6999bf6dd6501d8933575caab49827486ce774c6c35a6e411da8c6cfba",
+    "3ccfd4d8d625c708e593e46ea0404536984f69f11c7a08b5a59eb9bbc9dfde3c",
+    "dbcfa99f4d41f05e4b59eb2197873515c035eff1a68800bf0f43db471ef7ba57",
+    "56a43a2729035f7fec17b68823a7db9ea06fcf9ec8d0b5e70d73ab79488a1b21",
+    "b6b9d7c604cc55d2c6859ad098371dfbfe540eb2d5aab4faad21f81149659338",
+    "b7a3eee24d2b0910f3a07705f1e053221d2535f07c5a13614c4dead2620e9978",
+    "658cc9506acf89b5f5dfc6b55eb9899b782dcf8d89f3d11c9abb2c4d52617f12",
+    "21b43c2314614733f5e9b4b4f3117edc3986d2b0f2a33409b111cc38360e8391",
+    "aa1b09dca861a306cd24c2c47fded88b63ae833ff0e815dad3642465583e300f",
+    "2df9c9e54e5a2bc19c2ffd1df7ee283e02ba2e9a5fda1067b4884375ae4b0b7a",
+]
+options = ["empty"]
+
+
+def do_install(self):
+    for font in _fonts:
+        folder = font[2]
+        package = font[0]
+        self.install_file(
+            f"{folder}/*.*tf",
+            f"usr/share/fonts/nerd-{package}",
+            glob=True,
+        )
+        for file in [
```

Comment:
not sure if it's a good idea to install all the licenses separately for all fonts

Reply:
Well separate fonts have separate licenses, how would you do it instead?

Reply:
nevermind, i just misread what the code does - you have it right